bag_replicator/src/test/scala/uk/ac/wellcome/platform/archive/bagreplicator/replicator/ReplicatorTestCases.scala

```diff
@@ -299,46 +298,6 @@ trait ReplicatorTestCases[
     result.summary.maybeEndTime.isDefined shouldBe true
   }
 
-  // The verifier will write a Content-SHA256 checksum tag to objects when it
-  // verifies them.  If an object is then replicated to a new location, any existing
-  // verification tags should be removed.
```

Comment:
Never mind, I see where this test has gone.

~This test is in the replicator as a sort-of historical artefact; at one point you could choose whether you wanted to copy tags in the Transfer classes.~

~We rely on this behaviour to avoid the verifier tags getting copied from the source (and so skipping the verification in the post-replicator verifier).~

~Personally I would keep this test.~